partcad-cli/src/partcad_cli/click/commands/import.py

```
@@ -0,0 +1,48 @@
+import shutil
+import rich_click as click
+from pathlib import Path
+import partcad as pc
+from partcad.actions.part_actions import import_part_action
+
+@click.command(help="Import an existing part from another project and optionally convert its format.")
+@click.argument(
+    "part_type",
+    type=click.Choice(["stl", "step", "brep", "3mf", "obj", "gltf", "threejs"]),
+    required=True
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)